src/Modules/ParticleInCell/GravitySim.h

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
I think one thing that would help is to have a more descriptive name for this module, perhaps: ParticleSimulatorGravityField.  it would help distinguish it from the other test modules